web/client/plugins/manager/ManagerMenu.jsx

```diff
@@ -97,7 +95,18 @@ class ManagerMenu extends React.Component {
             .filter(e => this.props.enableContextManager || e.path !== "/context-manager")
             .sort((a, b) => a.position - b.position).map((entry) => {
                 return {
-                    action: (context) => {context.router.history.push(entry.path); return this.props.itemSelected(entry.id); },
+                    action: (context) => {
+                        context.router.history.push(entry.path);
+                        this.props.itemSelected(entry.id);
```

Comment:
have you checked that this action is called correctly?

Reply:
this works the same as before my changes